src/main/java/ai/applica/spring/boot/starter/temporal/processors/ActivityStubInterceptor.java

```diff
@@ -93,7 +93,9 @@ private ActivityOptions buildOptions(
 
     // Build default options
     Builder options = ActivityOptions.newBuilder();
-
+    if (!"".equals(activityStubAnnotation.taskQueue())) {
```

Comment:
You could use `StringUtils.hasText(activityStubAnnotation.taskQueue())` for readability.
Import is `import org.springframework.util.StringUtils;`

Reply:
OK